docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.7"
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: .docker/app/Dockerfile
+    stdin_open: true
+    tty: true
+    volumes:
+      - .:/app
+    depends_on:
+      - solr
+    ports:
+      - "3000:3000"
+    environment:
+      SOLR_URL: "http://solr:8983/solr/blacklight"
+      LC_ALL: "C.UTF-8"
```

Comment:
I don't understand why this is necessary.

Reply:
I can't remember why I ever did that so I'm removing it too.